package ru.itmo.exception

class UserAlreadyHasSuchCertificateException(message: String) : Exception(message)